packages/react-client/src/fishjamProvider.tsx

```diff
@@ -1,27 +1,45 @@
 import { useRef, useState, type PropsWithChildren } from "react";
 import { useTrackManager } from "./hooks/useTrackManager";
-import type { DeviceManagerConfig, PeerMetadata, ScreenShareState, TrackMetadata } from "./types";
+import type { DeviceManagerUserConfig, PeerMetadata, ScreenShareState, TrackMetadata } from "./types";
 import { FishjamClient, type ReconnectConfig } from "@fishjam-cloud/ts-client";
 import type { FishjamContextType } from "./hooks/useFishjamContext";
 import { FishjamContext } from "./hooks/useFishjamContext";
 import { DeviceManager } from "./DeviceManager";
 import { useParticipantStatus } from "./hooks/useParticipantStatus";
 import { useFishjamClientState } from "./hooks/useFishjamClientState";
+import { AUDIO_TRACK_CONSTRAINTS, VIDEO_TRACK_CONSTRAINTS } from "./constraints";
 
 interface FishjamProviderProps extends PropsWithChildren {
   config?: { reconnect?: ReconnectConfig | boolean };
-  deviceManagerDefaultConfig?: DeviceManagerConfig;
+  deviceManagerConfig?: DeviceManagerUserConfig;
```

Comment:
**suggestion:** user doesn't know anything about a device manager. the property could be just named `defaultConstraints` or `constraints`

Reply:
storage, if actually needed, could be an another property such as persistence options or sth

Reply:
WDYT about it?

```
interface FishjamProviderProps extends PropsWithChildren {
  config?: { reconnect?: ReconnectConfig | boolean };
  constraints?: Pick<MediaStreamConstraints, "audio" | "video">;
  persistLastDevice?: boolean | PersistLastDeviceHandlers;
}
```

Reply:
please unpack reconnect from config and i'm satisfied 💃

Reply:
updated ✅